### DISCUSSION TITLE
MacOS Ventura

### DISCUSSION THREAD

**Discussion by mantasmj** [ther0n/UnnaturalScrollWheels#68] [Q&A]
Hello,

I have just installed this app in hopes to solve everyday setting changing, when switching from magic mouse to trackpad, but it seems like it's not working in MacOS Ventura... :( 
Permissions were given - everything according to instructions, so is it Ventura incompatibility?

**Comment by Trezorro** [comment]
It works ike a charm for me on my Ventura 13.3.1 (a) (22E772610a) macbook pro (2019).
Thank you @ther0n @yan0f @AlexLiang27 and any other contributors!

**Reply by mantasmj** [reply]
Could you explain how you do it? I installed, gave permission in Privacy and Security -> Accessibility, kept the native scrolling setting "Natural", but it still doesn't invert. Nothing happens after reinstall either. I also have Ventura 13.3.1 -- how can it work for some users and not for the others? Clearly I've missed some important step.

**Comment by oleksii-molozhanov** [comment]
It suddenly stopped working for me on 13.5.2 (M1 Pro).
Before that was working like a charm for almost 2 years.

**Reply by mantasmj** [reply]
Was it ever working on M1 for you? I'm thinking maybe the Apple processor is the culprit why it never took off for me.

**Reply by oleksii-molozhanov** [reply]
It was working on my M1 for 2 years before without any issues. So probably it's not about the processor.
